ci/docker/docker-compose.yml

```diff
@@ -138,6 +138,8 @@ services:
         BASE_IMAGE: nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
       cache_from:
         - ${DOCKER_CACHE_REGISTRY}/build.ubuntu_gpu_cu110:latest
+    environment:
+      - CUDA_VERSION=11.0.3
```

Comment:
Please reference the upstream bug https://gitlab.com/nvidia/container-images/cuda/-/issues/92

Reply:
It turns out these environment variables are not visible during the build process, from what I can tell. Created a workaround in Dockerfile.build.ubuntu.

Reply:
Thanks